src/pixel.js

```diff
@@ -54,15 +62,22 @@ function createNoReferrerPixel(win, src) {
  * @param {!Window} win
  * @param {string} src
  * @param {boolean=} noReferrer
+ * @param {?boolean=} attributionReportingEligible
+ * @param {string=} attributionsrc
  * @return {!Image}
  */
-function createImagePixel(win, src, noReferrer = false) {
+function createImagePixel(win, src, noReferrer = false,
```

Comment:
if we do the check in this func I think we can get rid of  `attributionReportingEligible` param

Reply:
Done.